### DISCUSSION TITLE
blog/revert-purple/

### DISCUSSION THREAD

**Discussion by giscus** [ScratchAddons/website-v2#357] [Page comments]
# blog/revert-purple/

Here's how to switch from the new purple now used by the Scratch website and editor back to blue with Scratch Addons.

https://scratchaddons.com/blog/revert-purple/

**Comment by DNin01** [comment]
Ooh yeah the plans!

**Comment by mybearworld** [comment]
The purple is good :(

**Reply by CST1229** [reply]
I prefer blue.

**Reply by CatsMeow3377** [reply]
Me too

**Reply by BroJac5246** [reply]
I'm glad they added it (for accessibility), but I can't say I enjoy it.

**Reply by mybearworld** [reply]
I find that more contrast (to an extent) very often looks better, too.

**Comment by CatsMeow3377** [comment]
Yay! I hate the purple.
